src/vft.py

```diff
@@ -107,8 +107,19 @@ def generate_file(width, height, vft_id, tag_border_size, value, outfile, debug)
     cv2.imwrite(outfile, img)
 
 
-def analyze_file(infile, luma_threshold, debug):
+def analyze_file(infile, luma_threshold, width=0, height=0, debug=0):
     img = cv2.imread(cv2.samples.findFile(infile))
+    if width > 0 and height > 0:
+        dim = (width, height)
+    # Img stats
+    if debug > 0:
+        gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+        gmin = np.min(gray)
+        gmax = np.max(gray)
+        gmean = int(np.mean(gray))
+        gstd = int(np.std(gray))
+        print(f"min/max luminance: {gmin}/{gmax}, mean: {gmean} +/- {gstd}")
+    img = cv2.resize(img, dim, interpolation=cv2.INTER_AREA)
```

Comment:
This looks fishy: If the caller does not set both width and height, then `dim` is not set, and this line breaks. If you want to force the caller to set both width and height, do not set default values.